### PR TITLE
fixed adding-analytics.md

### PR DESCRIPTION
**Pull request by alexandrtovmach**
## Description

The main problem with documentation on GitHub is they are not showing filename, and I'm not sure that we can easily fix that, but anyway, file content really disappoint from the first look.

- changed file content
- removed duplicate note about replacing "with your own Tracking ID"
- added reference link to gatsby-config.js documentation page

## Related Issues

Related to #11957

### CODE REVIEW

File: docs/docs/adding-analytics.md [hunk 1]
```diff
@@ -37,16 +37,20 @@ npm install --save gatsby-plugin-google-analytics
 ```
 
 ```js:title=gatsby-config.js
+module.exports = {
+  plugins: [
     {
-      resolve: `gatsby-plugin-google-analytics`,
+      resolve: 'gatsby-plugin-google-analytics',
```
Comment (marcysutton) [inline docs/docs/adding-analytics.md]: This is fine, but as an FYI template literals are synonymous with strings.

File: docs/docs/adding-analytics.md [hunk 2]
```diff
@@ -37,16 +37,20 @@ npm install --save gatsby-plugin-google-analytics
...
       options: {
-      // replace "UA-XXXXXXXXX-X" with your own Tracking ID
+        // replace "UA-XXXXXXXXX-X" with your own Tracking ID
         trackingId: "UA-XXXXXXXXX-X",
       },
     },
+  ],
+};
```
Comment (marcysutton) [inline docs/docs/adding-analytics.md]: If you remove this semicolon, the linting will pass.

**Comment by marcysutton** [comment]
@alexandrtovmach thanks for working on this! It's really close to being ready to go but the linter is failing, and I can't check out the remote branch I think because the name has an invalid character in it (#). I ran linting locally and found the two issues were quotes (recommend leaving the template string in place), and an unneeded semicolon. If you can make those changes, I will merge.

**Comment by alexandrtovmach** [comment]
Great, thanks for recommendations, I'll implement it shortly.

File: docs/docs/adding-analytics.md (hunk 2 above)
Comment (alexandrtovmach) [reply docs/docs/adding-analytics.md]: fixed

File: docs/docs/adding-analytics.md (hunk 1 above)
Comment (alexandrtovmach) [reply docs/docs/adding-analytics.md]: fixed

**Comment by marcysutton** [comment]
Thanks for your contribution @alexandrtovmach! 🙌

**Comment by alexandrtovmach** [comment]
@marcysutton Thank you for help with my first steps in Gatsby contribution

**Pull request merged**